dev/tasks/python-wheels/github.windows.yml

```
@@ -43,6 +44,16 @@ jobs:
       {{ macros.github_login_ghcr()|indent }}
       {{ macros.github_install_archery()|indent }}
 
+      - name: Prepare
+        shell: bash
+        run: |
+          if [ "${PYTHON_ABI_TAG}" = "cp313t" ]; then
+            test_image_prefix=python-free-threaded
+          else
+            test_image_prefix=python
+          fi
```

Can we detect the last `t` for `free-threaded`? Python 3.14 will use `cp314t`, right?

```suggestion
          case "${PYTHON_ABI_TAG}" in
            *t)
              test_image_prefix=python-free-threaded
              ;;
            *)
              test_image_prefix=python
              ;;
          esac
```